TeltonikaMonitor/react/.env.production

```diff
@@ -0,0 +1,2 @@
+PUBLIC_URL=.
+DISABLE_ESLINT_PLUGIN = true
```

Comment:
co dela tohle? a proc to driv nebyl problem

Reply:
!